spyder/utils/icon_manager.py

```diff
@@ -133,7 +133,7 @@
     'breakpoint_transparent':  [('fa.circle',), {'color': 'darkred', 'opacity': 0.75, 'scale_factor': 0.9}],
     'breakpoint_big':          [('fa.circle',), {'color': '#cc0000', 'scale_factor': 0.9} ],
     'breakpoint_cond_big':     [('fa.question-circle',), {'color': '#cc0000', 'scale_factor': 0.9},],
-    'arrow_debugger':          [('mdi.arrow-right-thick',), {'color': '#3775a9'}],
+    'arrow_debugger':          [('mdi.arrow-right-thick',), {'color': '#3775a9', 'scale_factor': 2.0}],
```

Comment:
Let's change the arrow symbol to `mdi.arrow-right-bold`. I carefully compared the two arrows and I think this one looks better.